# HG changeset patch
# Parent 23fafa4a128bfb871a97a6e8dc9f470e7dc2ffb8
# User Jason Kim <jasonwkim@google.com>

Make EH stuff appear in .rodata. (Fixes arm failure)

diff -r 23fafa4a128b lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Tue Aug 09 16:42:05 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Tue Aug 09 16:42:05 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
